.github/workflows/deploy.yaml

```
@@ -0,0 +1,26 @@
+name: Deploy subgraph
+on:
+  push:
+    branches: ["main"]
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        chain: ["arbitrum-one", "arbitrum-rinkeby", "avalanche", "bsc", "celo", "fantom", "fuse", "goerli", "mainnet", "matic", "rinkeby", "xdai"]
+    environment: ${{ matrix.chain }}
```

Adding an URL makes the deployment clickable from the repo which could be convenient:
```suggestion
    environment: 
      name: ${{ matrix.chain }}
      url: https://thegraph.com/hosted-service/subgraph/requestnetwork/request-payments-${{ matrix.chain }}
```